superset-frontend/src/filters/components/Select/controlPanel.ts

```diff
@@ -93,7 +93,8 @@ const config: ControlPanelConfig = {
               resetConfig: true,
               affectsDataMask: true,
               renderTrigger: true,
-              description: t('Select first item by default'),
+              description: t('By default no filter is applied.' +
```

Comment:
"By default no filter is applied" can confuse the user because of the default filter option. I would suggest something like:
"If no default value is selected, this option will use the first available item as the default value."